pahfit/scripts/run_pahfit.py

```diff
@@ -70,6 +70,11 @@ def initialize_parser():
         type=int,
         help="Maximum number of interations for the fitting",
     )
+    parser.add_argument(
+        "--strength_calc",
+        action="store_true",
+        help="Calculate PAH feature and emission line strenghts",
+    )
```

Comment:
I would argue that computing the integrated feature strengths should be done by default, not only if the user requests.  This is not a computationally intensive calculation, so doing it by default will provide more information to users by default.  Hence this switch should be reversed to allow for the feature strengths to be turned off, not on.

Reply:
I totally agree with you. I only added an option to enable the calculation because it was mentioned at some point that we might not want to deliver measurements to the user. So since we are going to compute strengths by default, do we really need an option to override it? The output will include the feature strengths and the user can simply ignore them.

Reply:
Good point.  I would remove the option to overwrite it.  The user can just ignore it if they aren't interested.